cmd/commands.go

```diff
@@ -110,28 +107,19 @@ func New(plainLog, jsonLog log.Logger, conf config.Optimus, pluginRepo models.Pl
 	cmdx.SetHelp(cmd)
 	cmd.PersistentFlags().BoolVar(&disableColoredOut, "no-color", disableColoredOut, "Disable colored output")
 
-	// init local specs
-	datastoreSpecFs := make(map[string]map[string]afero.Fs)
-	for _, namespace := range conf.Namespaces {
-		dtSpec := make(map[string]afero.Fs)
-		for _, dsConfig := range namespace.Datastore {
-			dtSpec[dsConfig.Type] = afero.NewBasePathFs(afero.NewOsFs(), dsConfig.Path)
-		}
-		datastoreSpecFs[namespace.Name] = dtSpec
-	}
+	cmd.AddCommand(versionCommand(pluginRepo))
+	cmd.AddCommand(configCommand())
```

Comment:
We are going to remove this?

Reply:
config will be deleted alongside with this ticket https://github.com/odpf/optimus/issues/204